app/views/admin/questions/show.html.erb

```diff
@@ -10,7 +10,9 @@
 <div class="px-4">
   <h5>Title</h5>
   <div class="text-lg mb-1"><%= @question.title %></div>
-  <div class="text-sm mb-4"><%= @question.subtitle %></div>
+
+  <h5 class="mt-3">Subtitle</h5>
+  <div class="mb-4"><%= @question.subtitle %></div>
```

Comment:
Same removal action here as well

Reply:
The subtitle fields you have mentioned are on Questions pages not on Blogs Page. It is already added by you on questions pages but not properly styled. I just styled them.